### PR TITLE
Add run occ module

### PR DESCRIPTION
**Pull request by aalaesar**
Branch new module for running raw occ commands.
Also prepare foundation code for all other modules using occ

### CODE REVIEW

**Comment by aalaesar** [comment]
oops. still have to change/remove Licensing headers

File: plugins/module_utils/occ.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2022, Marc Crébassa <aalaesar@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+import os
+# import json
```
Comment (staticdev) [inline plugins/module_utils/occ.py:7]: Better to not have commented code on main branch.

File: plugins/module_utils/occ.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+
+def run_occ(module, command, ):
```
Comment (staticdev) [inline plugins/module_utils/occ.py:10]: Formatting of this file is very odd. Try to run black in whole folder, also useful to have on pre-commit.

[.pre-commit-config.yaml](https://github.com/psf/black/blob/main/.pre-commit-config.yaml):

```
---
repos:
  - repo: https://github.com/psf/black
    rev: 22.12.0
    hooks:
      - id: black
        language_version: python3.11
```

**Review by staticdev — CHANGES_REQUESTED** [review]
Some suggestions for QA.

File: plugins/module_utils/occ.py (hunk 1 above)
Comment (aalaesar) [reply plugins/module_utils/occ.py]: cleaned up  the commented line

File: plugins/module_utils/occ.py (hunk 2 above)
Comment (aalaesar) [reply plugins/module_utils/occ.py]: just done it.
how about now ? 😃

**Comment by aalaesar** [comment]
I'll try go see if I can add some basic [unit testing ](https://docs.ansible.com/ansible/latest/dev_guide/testing_units_modules.html) in the next couple of days

**Review by staticdev — APPROVED** [review]
LGTM. We have a basis to add tests and other code after that.

**Pull request merged**